flake.nix

```diff
@@ -1,4 +1,15 @@
 {
+  nixConfig = {
```

Comment:
I'd like this to be at the bottom of the file, but I think it definitely doesn't matter.

Reply:
Sure. Or I could make it alpabetical.

Reply:
Made it alphabetical, that OK?